README.md

```diff
@@ -3,26 +3,26 @@
 The OpenJS Foundation hosts regular summits for active contributors to the foundation projects, this repository is being used to organize them.
 
 ## Upcoming Events
-- [June 22, 25 & 26, 2020 - ~Austin, Texas, USA~ Online](https://github.com/openjs-foundation/summit/issues/236) - [Register Here](https://www.cvent.com/events/openjs-collaborator-summit-2020/registration-71e4281e54dd46a0a0e2da15013e52db.aspx?_ga=2.164120568.674445174.1588801738-1126120395.1571424999&fqp=true) or [Register for OpenJS World](https://events.linuxfoundation.org/openjs-world/)
+- [June 9, 10, 2022 - Austin, Texas, USA](https://github.com/openjs-foundation/summit/issues/310)
 
-__Note:__ The Collab Summit is now an online-only event, per the current [Novel Coronavirus Update](https://events.linuxfoundation.org/openjs-world/attend/novel-coronavirus-update/).
-
-To better adjust for a worldwide, online event we have added an additional day of programming. 
+## Registration
+- You can sign-up for Collab Summit when [registering for OpenJS World](https://events.linuxfoundation.org/openjs-world/)
+- If you are not attending OpenJS World 2022, and just want to register for Collab Summit, please reach out to [EMAIL].
 
 ## Planning Meetings
 
 Every 2 weeks on Tuesday - see [issues](https://github.com/openjs-foundation/summit/issues) for meeting notices and agendas
 
-6:00 – 7:00pm UTC 
+15:00 – 16:00 UTC
```

Comment:
Is this time good for everyone?

Reply:
Every other week, I have a half hour conflict on the bottom of the hour, but otherwise this time works for me, if I'm doing timezone translations correctly... We may want to do a Doodle poll on this though.

Reply:
Good point. Can you create a poll? I tried but couldn't figure it out.

Reply:
doodle.com -- sorry, a bit swamped right now and can't get to it at the moment.

Reply:
np, I have removed this section for now. We can add it later. :)